source/sass_api_reference/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Sass API Reference
+weight: 65
+---
+
+# Sass API Reference
+
+<% format_sassdoc_data(data).each do |heading, group| %>
+
+## <%= group_heading(heading) %>
+
+<% group.each do |doc| %>
+
+### <%= doc_heading(doc) %>
+
+<%= doc.description %>
+
+<% if doc.parameter %>
+| Name | Description | Type | Default value |
+| ---- | ----------- | ---- | ------------- |
+<% parameters_table(doc.parameter).each do |param| %>
+|  <%= param.name %> | <%= param.description %> | <%= param.type %> | <%= param.default_value %> |
+<% end %>
+<% end %>
+
+#### Usage
+
+<% if doc.deprecated %>
+    <% partial(:warning_text) do %>
+        <%= markdown(doc.deprecated) %>
+    <% end %>
+<% end %>
+
+<% if doc.parameter %>
+    <% inline_parameters = "(#{inline_parameters(doc.parameter)})" %>
+<% end %>
+
+<% if doc.context.type == 'mixin' %>
+```scss
+@include <%= doc.context.name %><%= inline_parameters %><%= mixin_trailing_code(doc.context.code) %>
+```
+<% end %>
+
+<% if doc.context.type == 'function' %>
+```scss
+$result: <%= doc.context.name %><%= inline_parameters %>;
+```
+<% end %>
+
+<% if doc.context.type == 'variable' %>
+```scss
+$<%= doc.type.downcase %>: $<%= doc.context.name %>;
+```
+<% end %>
+
+<% if doc.example %>
+#### Example
+
+<% doc.example.each do |example| %>
+##### <%= example.description %>
+
+```<%= example.type %>
+<%= example.code %>
+```
+<% end %>
+<% end %>
+
+View source code on GitHub: <a href="<%= github_url(doc) %>"><%= doc.file.name %></a>
```

Comment:
Is 'View source code on GitHub' better as the link text for this?

Reply:
I went with `View source code for <FILENAME> on GitHub`